app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -198,7 +198,6 @@ class RecentlyPlayedFragmentPresenter @Inject constructor(
             parentScreen = ExplorationActivityParams.ParentScreen.PARENT_SCREEN_UNSPECIFIED,
             isCheckpointingEnabled = canHavePartialProgressSaved
           )
-          activity.finish()
```

Comment:
Not sure why this needed to b e removed?

Reply:
I think that change is not needed.